api/AudioProcessingEvent.json

```diff
@@ -71,16 +71,16 @@
               "version_added": false
             },
             "opera": {
-              "version_added": null
+              "version_added": "44"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "43"
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
Seems right, this was just landed in WebKit: https://bugs.webkit.org/show_bug.cgi?id=215237